README.md

```diff
@@ -103,6 +103,17 @@ To translate between LLVM IR and SPIR-V:
     * `-spirv-text` - read/write SPIR-V in an internal textual format for debugging purpose. The textual format is not defined by SPIR-V spec.
     * `-help` - to see full list of options
 
+4. Multiple input files mode. Execute the following command to translate `input.bc` and `input1.bc` to `input.spv` and `input1.spv`.
+    ```
+    llvm-spirv input.bc input1.bc -o list.txt
```

Comment:
Will `list.txt` be generated by default, i.e. without `-o` option?

Reply:
Line 110 suggests that `-o` absence will result in just `input.spv`, `input1.spv`. But I agree that this behavior should be documented explicitly

Reply:
Documented explicitly.